frontend/src/components/ApprovalRequests.vue

```diff
@@ -1,57 +1,152 @@
 <template lang="pug">
-  v-container
-    v-row
-      v-col(cols='12', sm='3')
-        v-overflow-btn.my-2(:items='dropdown_item', label='Course Name', target='')
-    v-data-table.elevation-1(v-model='selected', :headers='headers', :items='ta', :single-select='singleSelect', item-key='name', show-select='')
     div
-      | By clicking Approve,you are certifying that the above student(s) have successfully performed the ta duty equivalent to * hours per week.
-    .text-center
-      v-row
-        v-col(cols='12', sm='2')
-          v-btn Approve
-        v-col(cols='12', sm='2')
-          v-btn Not Approved
+        v-data-table.elevation-1(
+            v-model="selected"
+            :headers='headers'
+            :items='currentApprovals',
+            show-select
+            sort-by='date_submitted')
+            template(v-slot:top='')
+                v-toolbar(flat='', color='white')
```

Comment:
This can be `v-toolbar(flat color='white')`, similarly `template(v-slot:top)` in above line.